src/sentry/integrations/slack/threads/activity_notifications.py

```diff
@@ -161,17 +167,30 @@ def get_ticket_number(self) -> str:
         return last_part
 
 
+class _GithubExternalIssueCreatedActivity(_ExternalIssueCreatedActivity):
+    """
+    Override class for Github, as the provider name that we want to display should be GitHub, not "Github"
+    """
+
+    def get_formatted_provider_name(self, _: str) -> str:
+        return "GitHub"
+
+
 def _external_issue_activity_factory(activity: Activity) -> _ExternalIssueCreatedActivity:
     """
     Returns the correct ExternalIssueCreatedActivity class based on the provider.
     All classes have the same interface, the method for one is simply modified for its use case.
     """
+    activity_classes = {
+        "asana": _AsanaExternalIssueCreatedActivity,
+        "github": _GithubExternalIssueCreatedActivity,
+    }
```

Comment:
feel free to move this out into a private global variable so it only initializes once on module load and saves some cpu/memory